example-gto/code/mlem/deploy/dev.mlem

```diff
@@ -0,0 +1,4 @@
+object_type: deployment
+type: heroku
+model: models/churn.pkl
+app_name: gto-example-mlem-app-dev
```

Comment:
Q: do we reserve this names, people won't be able to reuse them, right?

Reply:
Yes, and I guess we need to reserve `-staging` and `-prod` as well, in case we'll need them in future.

Reply:
add newline ... is it MLEM that is still producing this this way?

Reply:
No, it isn't. These are old files I modified manually, that's why we had it here.